pkg/cli/cmds/server.go

```diff
@@ -260,6 +262,11 @@ func NewServerCommand(action func(*cli.Context) error) cli.Command {
 				Usage:       "(components) Disable " + version.Program + " default network policy controller",
 				Destination: &ServerConfig.DisableNPC,
 			},
+			cli.StringSliceFlag{
+				Name:  "role",
+				Usage: "(components) Specify the role of the server process (valid items: controlplane etcd or controlplane and etcd)",
```

Comment:
I wonder if it'd be a benefit to the user if the "valid items" section was listed in the format expected. ie "controlplane", "etcd", or "controlplane,etcd".